model/metrics/container.yaml

```
@@ -11,7 +11,21 @@ groups:
     unit: "s"
     attributes:
       - ref: container.cpu.state
-        brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
+        brief: "The CPU state for this data point. A container's cpu time SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
+        requirement_level: opt_in
+
+  - id: metric.container.cpu.usage
+    type: metric
+    metric_name: container.cpu.usage
+    stability: experimental
+    brief: "Container's CPU usage, measured in cpus. Range from 0 to the number of allocatable cpus"
+    note: >
+      CPU usage of the specific container on all available CPU cores, averaged over the sample window
+    instrument: gauge
+    unit: "{cpu}"
+    attributes:
+      - ref: container.cpu.state
+        brief: "The CPU state for this data point. A container's cpu usage SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
```

maybe (to make it a bit more clear, also feel free to move it back to brief if you like it better)
```suggestion
        brief: "The CPU state for this data point."
        note: >
          `container.cpu.state` SHOULD be reported consistently on all measurements if it's reported at all.
```

---

I'm fine with changing this, however we define it like that in [system](https://github.com/open-telemetry/semantic-conventions/blob/main/model/metrics/system-metrics.yaml#L12) and [process](https://github.com/open-telemetry/semantic-conventions/blob/main/model/metrics/process-metrics.yaml#L12) metrics. Do you think diverging them would be ok?